src/lozad.js

```diff
@@ -33,6 +33,19 @@ const onIntersection = load => (entries, observer) => {
   })
 }
 
+const getElementsBySelector = selector => {
+  if (typeof selector === 'string') {
+    return document.querySelectorAll(selector)
+  } else if (selector instanceof Node) {
+    return [selector]
+  } else {
+    throw new Error(
+      `selector should be of type string ex: '.lozad' or ` +
```

Comment:
this error description will increase library size, let's remove the else part completely for now and let developer rely upon documentation

Reply:
Or I can do something like 
```js
if ( nodeTypeCheck ) {
return selector; 
}

return document.querySelectorAll(selector)
``` 
this way.. querySelectorAll will take care of validating the selector to be a string rather than handling it ourselves.
What do you say ?

Reply:
I think that's the best solution @PezCoder. Early returns are always more simpler.

Reply:
@PezCoder perfect